relational_operators/TextScanOperator.cpp

```diff
@@ -234,11 +234,10 @@ void TextScanOperator::receiveFeedbackMessage(const WorkOrder::FeedbackMessage &
   }
 }
 
-
```

Comment:
I cannot find it from our code style. But I remember I learnt it from some code reviews.

We can ignore this for now, and I would propose to add this to our code style. Thanks!